packages/frontapp/src/app/components/CategorySearch/index.tsx

```diff
@@ -5,17 +5,17 @@ import { attributes } from '../FilterAttributes';
 import Button from '../Button';
 import { createSearchParams, useNavigate } from 'react-router-dom';
 
-const Filters = styled.div`
+export const Filters = styled.div`
   font-weight: 700;
   font-size: ${dimensions.xl_2};
   margin-bottom: ${dimensions.base_2};
 `;
-const Title = styled.h3`
+export const Title = styled.h3`
   font-weight: 700;
   font-size: ${dimensions.base};
   margin-bottom: ${dimensions.lg};
 `;
-const AttributeWrapper = styled.div`
+export const AttributeWrapper = styled.div`
```

Comment:
Let's remove these exports and do it more correctly to avoid the problems in future (refactor this)

Reply:
whats the better way to do this, if the ui is same?

Reply:
Already discussed on discord call